### DISCUSSION TITLE
Projects -- Work/Fun Categories: remove work, substitute fun

### DISCUSSION THREAD

**Discussion by wcmohr** [alshedivat/al-folio#1379] [Q&A] [answered]
I'm just out of a data science bootcamp and don't have any work related projects, so I would like to remove the "Work" section from the Projects page.  I would also like to substitute the "Fun" category with categories such as "Exploratory Data Analysis" and "Modeling" and "Natural Language Processing", etc.  Is there a way to accomplish this?  I've tried changing the category in the .md for each project, but it doesn't work and the website breaks and goes back to the un-customized version.  Any tips are very welcome!

---
layout: page
title: Reddit Classification
description: Use of grid-search in training NLP model to classify between r/Technology and r/Science subreddits 
img: assets/img/reddit.png
redirect: https://github.com/wcmohr/AmesHousing
importance: 3
category: fun
---

**Comment by wcmohr** [comment]
https://github.com/wcmohr/wcmohr.github.io is my pages repo, https://wcmohr.github.io/ is the deployed site

**Comment by heyudude** [comment, marked as answer]
There is a projects.md file in _pages where you can add or remove categories:
![image](https://github.com/alshedivat/al-folio/assets/102884/b7828a8d-9300-4f41-9fae-e55f5aece39b)

Rebuild your jekyll instance "bundle exec jekyll build" if needed, hope this helps

**Reply by wcmohr** [reply]
Hey Dude, thanks this helped.  Didn't need to rebuild my jekyll.

**Reply by chriskcovert** [reply]
how do you rebuild your jekyll build if you are not working locally on your jekyll site and all publishing is done using Github Actions for Github pages?
